client/src/App.js

```diff
@@ -100,20 +100,20 @@ function App() {
             { /* All roles are authorised to access the front page, conditional rendering is done inside the component */ }
             <Route element={<PrivateRoute roles={[roles.admin, roles.teacher, roles.student, roles.assistant]}/>}>
               <Route path='/' element={<FrontPage/>} />
-              <Route path='/course-view/:courseCode' element={<CourseView/>}/>
+              <Route path='/course-view/:courseId' element={<CourseView/>}/>
             </Route>
             { /* Pages that are only authorised for admin */ }
             <Route element={<PrivateRoute roles={[roles.admin]}/>}>
               <Route path='/create-course' element={<CreateCourseView/>}/>
             </Route>
             { /* Pages that are authorised for admin and teachers */ }
             <Route element={<PrivateRoute roles={[roles.admin, roles.teacher]}/>}>
-              <Route path='/fetch-instances/:courseId' element={<FetchInstancesView/>}/>
+              <Route path='/fetch-instances/:courseId/:courseCode' element={<FetchInstancesView/>}/>
```

Comment:
Should this be changed to :courseId/fetch-instances/:courseCode so the paths are consistent

Reply:
I actually did this change `':courseId/fetch-instances/:courseCode'` in #173 so if this is wanted for this branch as well, examples can be looked from there.

Reply:
Well use your changes, I don't think the work should be duplicated